lib/system/error_reporting.rb

```diff
@@ -14,7 +14,7 @@ module ErrorReporting
     def report_error(exception, rack_env: nil, logger: Rails.logger, **parameters)
       options = OPTIONS.call(parameters, rack_env)
 
-      logger.error('Exception') { exception }
+      logger.error('Exception') { {exception: {class: exception.class, message: exception.message}, parameters: parameters} }
```

Comment:
I'd also log the backtrace somehow or at least where it was invoked

Reply:
The whole `backtrace` feels like too much spam:

![image](https://user-images.githubusercontent.com/11318903/49645288-9ffcb580-fa1b-11e8-8246-786981de195a.png)
![image](https://user-images.githubusercontent.com/11318903/49645304-a9861d80-fa1b-11e8-81a4-dc95afb6050b.png)

However, some thing like this seems more reasonable:
`logger.error('Exception') { {exception: {class: exception.class, message: exception.message, backtrace: exception.backtrace[0..3]}, parameters: parameters} }`
![image](https://user-images.githubusercontent.com/11318903/49645380-f79b2100-fa1b-11e8-8c53-346cbe9467b7.png)

WDYT @hallelujah ?

Reply:
![image](https://user-images.githubusercontent.com/11318903/49645610-eacafd00-fa1c-11e8-922e-05c34ac6c796.png)

Reply:
OK for the 4 traces